### ISSUE TITLE
Pushing multiple local commits only creates one entry in events_raw table

### ISSUE THREAD

**Issue by philipdalen** [GoogleCloudPlatform/fourkeys#123]
**Steps to reproduce**
1. Create multiple local commits 
2. Do one push to remote (This means that multiple commits get's pushed to remote at once)
3. Go to the events_raw table in GC and observe that only one entry has been made events_raw table. 
4. Observe that the commit_id the last commit of the batch of commits has been used as the id in the events_raw table
5. Observe that the other commits are only accessible in the metadata field.

The reason why this is a problem is that based on the sql query that generates the changes only the latest commit will show up in the changes table, since the query never look inside the metadata for additional commits
```sql
# Changes Table
SELECT 
source,
id as change_id,
time_created,
event_type,
metadata,
FROM four_keys.events_raw 
WHERE event_type in ("pull_request", "push", "merge_request") 
```

**Comment by dinagraves** [comment]
Ah thanks for finding this!  I'll see if I can recreate.  My first thought is that the changes script should be updated to unpack those other commits from the metadata.

**Comment by philipdalen** [comment]
Yes, that is also my understanding, I already create a PR for it here https://github.com/GoogleCloudPlatform/fourkeys/pull/132
